scripts/release_info.py

```diff
@@ -292,24 +303,111 @@ def _check_storage_models(current_release):
     return [item for item in diff_list if item.startswith('core/storage')]
 
 
+def check_blocking_bugs(repo):
+    """Checks the number of unresolved blocking bugs.
+
+    Args:
+        repo: github.Repository.Repository. The PyGithub object for the repo.
+
+    Returns:
+        int: Number of unresolved blocking bugs.
+    """
+    blocking_bugs_milestone = repo.get_milestone(number=39)
```

Comment:
Should we move this number and other such constants to feconf? (not sure whether we should use feconf for release script)

@seanlip, do you have any suggestion on this?

Reply:
I think it is fine to add it on top with a comment.

Reply:
In general, it is fine to keep constants at the top of a script if the script is the only file which uses them.

Reply:
Done.